connectors/jdbc/src/main/java/io/camunda/connector/jdbc/utils/ConnectionHelper.java

```diff
@@ -27,13 +29,33 @@ public static Connection openConnection(JdbcRequest request) {
       JdbcConnection connection = request.connection();
       Connection conn =
           DriverManager.getConnection(
-              connection.getConnectionString(request.database()), connection.getProperties());
+              ensureMySQLCompatibleUrl(
+                  connection.getConnectionString(request.database()), request.database()),
```

Comment:
nitpick: for readability and easier debugging later, consider defining `request.database()` as a variable earlier.

Reply:
good point, done